tests/float/Makefile

```diff
@@ -1,8 +1,3 @@
 include ../Makefile.tests_common
 
-# for native we can go do a couple of more operations in reasonable time...
-ifneq (,$(filter native,$(BOARD)))
-  CFLAGS += -DTEST_ITER=100000000
-endif
-
```

Comment:
Why not change the value for msp430 then? Why increase it for native in the first place?

Reply:
Why have many iterations in the first place? The test doesn't have that info. it is quite old. The linked bug report shows errors in 2 of 5 iterations. we don't know more.

If noone remembers why this needs to run for 100k iterations (or 100 million, on native), we can save the CI time.

With 1000 iterations, the test confirms that basic double compiler and library functionality is working. IMO that's totally sufficient.

Reply:
> Why have many iterations in the first place?

That is the exact question I'm asking hahaha :).